### PR TITLE
[DEV-5856] Elasticsearch ETL refactor (part 3)

### PR DESCRIPTION
**Pull request by tony-sappe**
**Description:**
Splitting large files into smaller files. Renamed `RapidLoader` to `Controller`

**Technical details:**
No behavior should have been modified by these changes

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
2. [x] API documentation updated (N/A)
3. [ ] Necessary PR reviewers:
    - [ ] Backend
4. [x] Matview impact assessment completed (N/A)
5. [x] Frontend impact assessment completed (N/A)
6. [x] Data validation completed
7. [x] Appropriate Operations ticket(s) created (N/A)
8. [x] Jira Ticket [DEV-5856](https://federal-spending-transparency.atlassian.net/browse/DEV-5856):
    - [x] Link to this Pull-Request
    - [x] Performance evaluation of affected Script (no changes)
    - [x] Before / After data comparison

**Area for explaining above N/A when needed:**
```
No impact to API or database
```

### CODE REVIEW

File: usaspending_api/etl/elasticsearch_loader_helpers/__init__.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+from usaspending_api.etl.elasticsearch_loader_helpers.fetching_data import (
+    configure_sql_strings,
+    download_db_records,
+    get_updated_record_count,
+)
+from usaspending_api.etl.elasticsearch_loader_helpers.delete_data import (
```
Comment (kwhickey) [inline usaspending_api/etl/elasticsearch_loader_helpers/__init__.py:6]: Just for consistency, you have `fetching_data` and `indexing_data` ... to follow suit this one should be `deleting_data`. Although I'm more partial to flipping the other two to singular (`index_data`, `fetch_data` along with `delete_data`)

Comment (kwhickey) [reply usaspending_api/etl/elasticsearch_loader_helpers/__init__.py:6]: Also, _could_ follow ETL semantics and be `extract_data`, `load_data`, `delete_data` ... 

And then `ElasticsearchRunner` might fit better if renamed to `ElasticsearchEtl`

File: usaspending_api/etl/elasticsearch_loader_helpers/fetching_data.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+"""
+
+# ==============================================================================
+# Other Globals
+# ==============================================================================
+
+AWARD_DESC_CATEGORIES = {
+    "loans": "loans",
+    "grant": "grants",
+    "insurance": "other",
```
Comment (kwhickey) [inline usaspending_api/etl/elasticsearch_loader_helpers/fetching_data.py:40]: Is that right? `insurance` goes under `other` right now?

**Review by kwhickey — APPROVED** [review]
Minor considerations on naming consistency. Otherwise looks fine as a code refactor.

File: usaspending_api/etl/elasticsearch_loader_helpers/__init__.py (hunk 1 above)
Comment (tony-sappe) [reply usaspending_api/etl/elasticsearch_loader_helpers/__init__.py:6]: Open to any and all naming suggestions.

File: usaspending_api/etl/elasticsearch_loader_helpers/fetching_data.py (hunk 2 above)
Comment (tony-sappe) [reply usaspending_api/etl/elasticsearch_loader_helpers/fetching_data.py:40]: For now. There is a desire to split it into it's own category in the future

File: usaspending_api/etl/elasticsearch_loader_helpers/__init__.py (hunk 1 above)
Comment (tony-sappe) [reply usaspending_api/etl/elasticsearch_loader_helpers/__init__.py:6]: @kwhickey Pushed a commit with new names. Didn't change `fetch` -> `extract` but I can if we still feel that is preferable.

**Comment by collinwr** [comment]
@tony-sappe if I understand this correctly, a single file was split out into a package with no changes to the contents of the methods?

**Review by sethstoudenmier — APPROVED** [review]
Local tests run as expected.

**Pull request merged**